src/components/bit-brush-mode/bit-brush-mode.jsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ToolSelectComponent from '../tool-select-base/tool-select-base.jsx';
+import {defineMessages} from 'react-intl';
 
 import brushIcon from './brush.svg';
 
+const messages = defineMessages({
+    brush: {
+        defaultMessage: 'Brush',
+        description: 'Label for the brush tool',
+        id: 'paint.brushMode.brush'
+    }
+});
+
```

Comment:
@apple502j this is looking really great. There's one slight issue - the same `id` should not be defined in multiple places. Currently the definitions are all the same, but it can lead to bugs later when only one gets changed. To fix this the messages that are shared between components should be pulled out into a shared file. Probably it should be something like `.../lib/messages.js`. It just needs to import `defineMessages` and export the messages object: 
```
import {defineMessages} from 'react-intl';

const messages = defineMessages({
    brush: {
        defaultMessage: 'Brush',
        description: 'Label for the brush tool',
        id: 'paint.brushMode.brush'
    },
    eraser: {
        defaultMessage: 'Eraser',
        description: 'Label for the eraser tool',
        id: 'paint.eraserMode.eraser'
    },
    etc....
});

export default messages;
```

Then in the component, just import messages from the shared file instead of defining the message.

To find duplicates you can run `npm run i18n:src` it will stop as soon as it finds one. Just remember to remove the temporary translations folder if it stops with an error.

Reply:
Thanks! I'll try it.